hazelcast/src/main/java/com/hazelcast/spi/impl/executionservice/impl/SchedulingExecutorServiceDelegate.java

```diff
@@ -27,12 +29,14 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 
-final class ScheduledExecutorServiceDelegate implements ScheduledExecutorService {
+import static com.hazelcast.util.Preconditions.checkNotNull;
+
+public final class SchedulingExecutorServiceDelegate implements SchedulingExecutorService {
```

Comment:
I would name this class SchedulingExecutorServiceImpl. It really isn't relevant that it is a delegate. Apart from that most services are called FooImpl.